exp-player/addon/components/exp-overview/component.js

```diff
@@ -29,6 +29,12 @@ var generateValidators = function(questions) {
   return validators;
 };
 
+var getQuestion8Options = function() {
+    var scale = range(1, 10);
+    scale.push('survey.sections.1.questions.8.options.preferNoAnswer');
```

Comment:
Good question! According to their code book, "I prefer not to answer" is represented by the number 11.